examples/landing-zone-v2/tier3/external-load-balancer/elb.yaml

```diff
@@ -0,0 +1,27 @@
+# Copyright 2021 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#########
+# create the external load balancer
```

Comment:
can you double check that statement. To me, a URLmap is a component of a load balancer. I think the forwardingrule is the top resources that could be defined as the load balancer..but I'm not 100% sure

Reply:
not 100% sure either why this component is used to represent the "LOAD BALANCERS" in the console...

![image](https://github.com/GoogleCloudPlatform/pubsec-declarative-toolkit/assets/94859304/6e3c0399-c39e-4542-a21e-019a00b69ac4)

also used in other sections of the web console, like when adding a target to a cloud armor policy, etc...

![image](https://github.com/GoogleCloudPlatform/pubsec-declarative-toolkit/assets/94859304/5f4c8071-d9bc-41c9-847b-e44d70ba80c1)

Reply:
good to know. thank you for the validation